api/test/zero/unit/aou_module_test.clj

```diff
@@ -23,43 +24,49 @@
                                             :zcall_thresholds_file       "foo"}
         redundant-per-sample-inputs-inputs (merge expected-per-sample-inputs {:extra "bar"})
         missing-per-sample-inputs-inputs   (dissoc expected-per-sample-inputs :analysis_version_number)
-        all-expected-keys                  [:Arrays.preemptible_tries
-                                            :Arrays.environment
-                                            :Arrays.ref_dict
-                                            :Arrays.params_file
-                                            :Arrays.subsampled_metrics_interval_list
-                                            :Arrays.chip_well_barcode
-                                            :Arrays.sample_alias
-                                            :Arrays.control_sample_vcf_file
-                                            :Arrays.variant_rsids_file
-                                            :Arrays.ref_fasta_index
-                                            :Arrays.dbSNP_vcf
-                                            :Arrays.disk_size
-                                            :Arrays.contamination_controls_vcf
-                                            :Arrays.green_idat_cloud_path
-                                            :Arrays.fingerprint_genotypes_vcf_index_file
-                                            :Arrays.control_sample_intervals_file
-                                            :Arrays.vault_token_path
-                                            :Arrays.reported_gender
-                                            :Arrays.dbSNP_vcf_index
-                                            :Arrays.extended_chip_manifest_file
-                                            :Arrays.zcall_thresholds_file
-                                            :Arrays.sample_lsid
-                                            :Arrays.red_idat_cloud_path
-                                            :Arrays.gender_cluster_file
-                                            :Arrays.control_sample_vcf_index_file
-                                            :Arrays.genotype_concordance_threshold
-                                            :Arrays.ref_fasta
-                                            :Arrays.bead_pool_manifest_file
-                                            :Arrays.analysis_version_number
-                                            :Arrays.fingerprint_genotypes_vcf_file
-                                            :Arrays.control_sample_name
-                                            :Arrays.cluster_file
-                                            :Arrays.call_rate_threshold
-                                            :Arrays.haplotype_database_file]]
```

Comment:
Why `sort`?

Reply:
since it seems the order matters with `[]`

Reply:
Order matters for `seq`s but not for `set`s.  Hence my comment below.

Reply:
Fair enough, changing them to `set` now.